### ISSUE TITLE
Potential small bug in travis scoring

### ISSUE THREAD

**Issue by craigjmcgowan** [FluSightNetwork/cdc-flusight-ensemble#61]
I've finished checked the scores generated in travis against scores calculated using the FluSight R package. We're down to 110 discrepancies of greater than 10^-12, all related to peak week in the 2014/15 season. The errors occur in Regions 2, 3, 5, and 7, as well as US National, all of which have week 52 as the peak week. 

I looked at one particular error in detail - the target-based model forecasts for Epiweek 53. Specific target is HHS Region 3 peak week. Correct score should be -0.627, summing probabilities of weeks 51, 52, and 53. Travis is assigning -1.11, apparently from summing probabilities of weeks 51, 52, and 1.

![image](https://user-images.githubusercontent.com/20864042/32076417-e69a4672-ba6d-11e7-9451-88d58edca6f4.png)

**Comment by lepisma** [comment]
I will look into this. Though this makes me wonder if we should try to fix the js implementation or use the FluSight package for scoring directly (since R is running on travis now). That will have a few benefits:
1. More verified/tested scoring
2. Items in our repo will get more standardized like truth files etc.
3. We will get upstream changes from FluSight package.

**Comment by lepisma** [comment]
I will look into this. Though this makes me wonder if we should try to fix the js implementation or use the FluSight package for scoring directly (since R is running on travis now). That will have a few benefits:
1. More verified/tested scoring
2. Items in our repo (like truth files etc.) will get more standardized
3. We will get upstream changes from FluSight package

**Comment by craigjmcgowan** [comment]
If using the FluSight package is feasible without too much work we could do that. I've never worked with Travis but could give it a shot to implement. I can try and read through the js file and create a similar implementation in R. We'd want a script that would work on each folder of submissions, correct?

**Comment by lepisma** [comment]
Yes, we need to sweep through the directories `model-forecasts/component-models` and `model-forecasts/cv-ensemble-models`. There is a csv mapping a list of _complete_ component models [here](https://github.com/FluSightNetwork/cdc-flusight-ensemble/blob/master/model-forecasts/component-models/model-id-map.csv).

By the way, I fixed an issue with counting number of mmwr weeks in a season. Looking at the scores, I can confirm fixes to the mentioned week 53 issue. I also ran the check-ensemble-scores and found no diff this time (0 obs in `different_score <- compare_scores %>% filter(diff > 1e-12)`). If thats fine then we can keep switching to R a low priority task.

Currently running the check-ensemble-scores script on all the models (not only ensemble models). Will write back if there is any diff there.
